utils/datetime_utils.py

```diff
@@ -69,7 +69,7 @@ def datetime_to_unix_time(dt):
 
 
 def unix_time_to_datetime(timestamp, tzinfo=pytz.utc):
-    dt = datetime.datetime.fromtimestamp(timestamp).replace(tzinfo=tzinfo)
+    dt = datetime.datetime.fromtimestamp(timestamp, tz=tzinfo)
```

Comment:
- Can you rebase off of `master` and delete the other `unix_time_to_utcdatetime` ? https://github.com/hacktoolkit/django-htk/blob/master/utils/datetime_utils.py
- Does this work for both Python 2 and 3?
- Was the goal of the function to return a timezone-aware `datetime.datetime` object? If so, I think we still need to keep the `.replace(...)` call

Reply:
- Done.
- It works for both Python2 and Python3.
- This itself returns `timezone-aware datetime`. 

Attached images for reference,
![py2](https://github.com/hacktoolkit/django-htk/assets/80397858/8a26c67b-a69d-4db4-a4ce-c2772baaf8b8)

![py3](https://github.com/hacktoolkit/django-htk/assets/80397858/3b1fc76a-559a-4fe2-92ef-76d0ce28406c)